roles/keycloak_quarkus/handlers/main.yml

```diff
@@ -7,7 +7,8 @@
   ansible.builtin.include_tasks: bootstrapped.yml
   listen: bootstrapped
 - name: "Restart {{ keycloak.service_name }}"
-  ansible.builtin.include_tasks: restart.yml
+  ansible.builtin.include_tasks:
+    file: "{{ keycloak_quarkus_restart_strategy if keycloak_quarkus_ha_enabled else 'restart.yml' }}"
   listen: "restart keycloak"
```

Comment:
the linter seems to prefer `Restart keycloak` here

Reply:
That's new.. and kind of non-sense to me https://github.com/ansible/ansible-lint/issues/4168